src/lib/settings/SettingsFolder.js

```diff
@@ -101,6 +101,23 @@ class SettingsFolder extends Map {
 		return this.relative(pathOrPiece.path);
 	}
 
+	/**
+ * Resolves paths into their full objects or values depending on the current set value
+ * @since 0.5.0
+ * @param  {...string} paths The paths to resolve
+ * @returns {*}
+ */
+	async resolve(...paths) {
+		const resolved = {};
+		const guild = resolveGuild(this.base.gateway.client, this.base.target);
+		const language = guild.language || this.client.languages.default;
+		for (const path of paths) {
+			const piece = this.schema.get(path);
```

Comment:
this.relative resolves SchemaPieces and absolute paths into relative paths to this schema folder
```suggestion
			const piece = this.schema.get(this.relative(path));
```